services/common/src/interfaces/mineIncident.interface.ts

```diff
@@ -0,0 +1,6 @@
+export interface IMineIncident {
+  mine_name: string;
+  categories: any;
+  incident_timestamp: any;
+  mine_incident_guid: string;
```

Comment:
You're probably going to have to make some adjustments with the timezone/timestamp now, once you pull those changes. :grin: 
The incident_timezone will be a string, and there are some values in strings.js now that should be the possible values if you wanted to go there (making a timezone type is probably scope creep). The timestamp will be an ISO string.

Reply:
Resolved the merge conflicts in `MineIncident.tsx` file.